tests/AntiMattr/Tests/MongoDB/Migrations/Configuration/ConfigurationTest.php

```diff
@@ -160,7 +160,7 @@ public function testGetNumberOfExecutedMigrations()
             ->with('antimattr_migration_versions_test')
             ->willReturn($collection);
 
-        $cursor = $this->createMock('Doctrine\MongoDB\Cursor');
+        $cursor = $this->createMock('MongoDB\Driver\Cursor');
```

Comment:
this cursor is final so can not be mocked. Any suggestions on how to update these tests?

Reply:
Replaced with a cursor mock, open to other suggestions

Reply:
It should be enough for now.